Add nucleotides, fix CONTRIBUTING typos

emleddin opened the pull request:
I created some nucleotide blocks with the letters in different rotations, and I'm including them under CC0. Please let me know if I need to fix anything!

duerrsimon reviewed (changes requested):
Thanks Emmet. Please remove the automatically generated files from the request and only include the `*.md` and `*.svg` files in your pull request. 

Also it would be good to only include one icon per nucleotide and not all rotated copies. The point of making them available as vector is that such minor modifications can be quickly made in the vector editor and it would bloat the platform a bit with icons that are not substantially different.

Thank you for contributing to Bioicons.

emleddin commented:
I did some resets, and I believe the lock and json files are gone now. I added some other icons to the PR and removed the nucleotide rotations (but kept the blanks).

duerrsimon commented:
Hi Emmet, this has been on my todo list for ages and I wanted to combine it with some other icons to add. I'm currently finishing up a research stay in Stanford and will get back to this once back in Switzerland. Sorry to keep you waiting.
